### DISCUSSION TITLE
im kinda new

### DISCUSSION THREAD

**Discussion by angelfyre1618** [community/community#48652] [New to GitHub] [answered]
how does this work?

**Comment by stevepopovich** [comment]
Hi @angelfyre1618, welcome.  This is the public GitHub feedback form. You can submit feedback or questions or bug reports. This category is specific to the native GitHub mobile app. Does that help at all?

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to GitHub and the GitHub Community [angelfyre1618](https://github.com/angelfyre1618) 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
